AutomatedLabCore/functions/VirtualMachines/Get-LabVM.ps1

```diff
@@ -124,6 +124,12 @@
             {
                 $machine.Disks = Get-LabVHDX -Name $machine.Disks.Name -ErrorAction SilentlyContinue
             }
+
+            $writeVmConnectConfigFile = Get-LabConfigurationItem -Name VMConnectWriteConfigFile
+            if ($writeVmConnectConfigFile)
+            {
+                New-LWHypervVmConnectSettingsFile -VmName $vm.Name
```

Comment:
`New-LWHypervVmConnectSettingsFile -VmName $machine.Name`

Reply:
I have removed these lines altogether, the files are only written when creating the VM. If one needs to update the config files, this line helps:

```powershell
Get-LabVM | ForEach-Object { New-LWHypervVMConnectSettingsFile -VmName $_ }
```

Do you think creating the files during the creation process is fine or should be regularly update them?